### ISSUE TITLE
Each doc should contain a link to it's github source/path

### ISSUE THREAD

**Issue by PaulBone** [PlasmaLang/plasma#90]
Each doc should contain a link to it's github source/path.  This will make it easier for users who notice minor problems to say "Oh, I can fix it" and find the right file easily.

**Comment by PaulBone** [comment]
Not adding the asciidoc skill, because the single item needed to do this is easy to learn. I wouldn't consider it a prerequisite.

**Comment by jacebenson** [comment]
Would this be as simple as adding something like;
```
link:https://github.com/PlasmaLang/plasma/tree/master/docs/plasma_ref.html[Improve this page]
```
on each of the `.txt` files in here; https://github.com/PlasmaLang/plasma/tree/master/docs?

**Comment by PaulBone** [comment]
Yep, something like that, in the appropriate spot in each document.

Cheers.

**Comment by PaulBone** [comment]
Actually it ends in .txt but you got the idea.

**Comment by PaulBone** [comment]
This has been merged as 941cb07e58a3ab00204590103c8a1f7f50588389 and #92 Thanks @jacebenson.
